monitoring/grafana-datasource-es.yaml

```diff
@@ -15,5 +15,6 @@ datasources:
     logMessageField: message
     logLevelField: level
   secureJsonData:
-    basicAuthPassword: admin
+    # Change this value
+    basicAuthPassword: YOUR_PASSWORD_HERE
```

Comment:
Is this value still used?  Looks like they either have 2 different ways of setting it or need to set it twice.  
If the env var is the right way (or over-rides the value here) I would comment out this line and change the comment to say "_This value isn't used, set the admin password by setting the GRAFANA_ADMIN_PASSWORD environment variable._"

Reply:
It's not used by default. Setting ELASTICSEARCH_DATASOURCE=true will deploy the Elasticsearch datasource. If enabled, they will need to set the password. The password is the Elasticsearch password, not the Grafana password.